specs/latest/2.0/index.html

```diff
@@ -1063,8 +1063,17 @@ <h2 class="no-toc">Table of contents</h2>
     <h4>Framebuffer objects</h4>
 
     <dl class="methods">
-      <dt class="idl-code">any getFramebufferAttachmentParameter(GLenum target, GLenum attachment,
-                                        GLenum pname)
+      <dt class="idl-code">[WebGLHandlesContextLoss] GLenum checkFramebufferStatus(GLenum target)
+          <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.3.pdf#nameddest=section-4.4.5">OpenGL ES 3.0.3 &sect;4.4.4</a>, <a class="nonnormative" href="http://www.khronos.org/opengles/sdk/docs/man3/html/glCheckFramebufferStatus.xhtml">man page</a>)</span>
+      <dd>
+        <p>Only differences from <a href="https://www.khronos.org/registry/webgl/specs/latest/1.0/#CHECK_FRAMEBUFFER_STATUS">checkFramebufferStatus</a> in WebGL 1 are described here.</p>
```

Comment:
I'm pretty sure this is the first reference from the WebGL 2.0 to the 1.0 spec. It's a little complicated to handle these because theoretically we should use relative URLs. However this seems fine as a start. Please give a little more thought to how we might handle these references better.